src/dsarchive.h

```diff
@@ -11,6 +11,7 @@
 #define SDAYLAYOUT  "%n.%s.%Y:%j"
 #define BUDLAYOUT   "%n/%s/%s.%n.%l.%c.%Y.%j"
 #define CSSLAYOUT   "%Y/%j/%s.%c.%Y:%j:#H:#M:#S"
+#define SDSLAYOUT   "%Y/%n/%s/%c.D/%n.%s.%l.%c.D.%Y.%j"
```

Comment:
For completeness, I note that this forces all output to the 'D' (waveform data) type, which is probably OK since that typing is limited in usefulness anyway.